Common/Core/TrackSelectionDefaults.cxx

```diff
@@ -76,4 +79,32 @@ TrackSelection getGlobalTrackSelectionSDD()
   return selectedTracks;
 }
 
+// Default track selection for nuclei analysis in run3 (STILL JUST A PLACEHOLDER)
+TrackSelection getGlobalTrackSelectionRun3Nuclei()
+{
+  return getGlobalTrackSelection();
+}
+
+// Default track selection for HF analysis (global tracks, with its points, but no tight selection for primary) in run3 (STILL JUST A PLACEHOLDER)
+TrackSelection getGlobalTrackSelectionRun3HF()
+{
+  TrackSelection selectedTracks;
+  selectedTracks.SetTrackType(o2::aod::track::Run2Track); // Run 2 track asked by default
```

Comment:
Maybe this could be Run3 ?

Reply:
yes, that's a good point, I'll change it